tests/endtoend_tests/test_examples.py

```diff
@@ -726,7 +726,7 @@ def test_unit_tesseract_endtoend(
             return
 
         if unit_tesseract_config.test_with_random_inputs:
-            payload_from_schema = {"inputs": example_from_json_schema(out_input_schema)}
```

Comment:
How come the tests passed without this?

Reply:
It's an expected change for this pr; before `/input_schema` returned a dump of `InputSchema`, not `ApplyInputSchema`